cpp/doAll_Zp.C

```diff
@@ -29,6 +29,12 @@
   map<TString,TString> sample_names = { };
   map<TString,map<TString,vector<TString>>> sample_prod = { };
 
+  bool run_data = 0;
+  bool run_MCbkg = 0;
+  bool run_signal = 1;
+//  bool pick_signal_mass = 0;
```

Comment:
Maybe remove the comment, since it is not used anywhere?

Reply:
fixed